src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala

```diff
@@ -84,6 +84,7 @@ object BookApiProperties extends LazyLogging {
   lazy val MetaSchema = prop(PropertyKeys.MetaSchemaKey)
   val MetaInitialConnections = 3
```

Comment:
Blir denne brukt etter denne endringen eller kan den slettes?

Reply:
Fra dok til Hikari:
However, for maximum performance and responsiveness to spike demands, we recommend not setting this value and instead allowing HikariCP to act as a fixed size connection pool.

Reply:
Den kan altså fjernes?

Reply:
Jepp. Tar den bort.